### PR TITLE
[full-ci][tests-only] Adding spaces webdav variant for favorites api features

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR adds tests related to spaces WebDAV in apiFavorite folder.

## Related Issue

- https://github.com/owncloud/ocis/issues/2854

## How Has This Been Tested?

- Locally
- CI
- https://github.com/owncloud/ocis/pull/2943

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

### CODE REVIEW

**Review by SwikritiT — COMMENTED** [review]
Please add the `ocis` PR adjusting the expected to fail file in `How Has This Been Tested?` section 
Otherwise looks good to me

**Review by kiranparajuli589 — COMMENTED** [review]
looks good :+1:

**Review by Talank — COMMENTED** [review]
LGTM

**Comment by kiranparajuli589** [comment]
test pr build has passed https://drone.owncloud.com/owncloud/ocis/8768

**Review by individual-it — APPROVED** [review]

**Comment by individual-it** [comment]
please bump the commit id in ocis to make sure that gets in ASAP

**Comment by individual-it** [comment]
unit tests failed with
```
+ wait-for-it -t 600 fsweb.test.owncloud.com:445
services aren't ready in 10m0s
```
restarted

**Review by phil-davis — APPROVED** [review]

File: tests/acceptance/features/apiFavorites/favorites.feature [hunk 1]
```diff
@@ -112,7 +137,12 @@ Feature: favorite
       | old         |
       | new         |
 
-  @files_sharing-app-required
+    @skipOnOcV10
+    Examples:
+      | dav_version |
+      | spaces      |
+
+  @files_sharing-app-required @issue_ocis_2968
```
Comment (SwikritiT) [inline tests/acceptance/features/apiFavorites/favorites.feature:145]:
```suggestion
  @files_sharing-app-required @issue-ocis-2968
```
I think we write it like this

**Comment by kiranparajuli589** [comment]
php unit suites failed with 
> + wait-for-it -t 600 fsweb.test.owncloud.com:445
> services aren't ready in 10m0s

build restarted!

**Pull request merged**